sympy/core/mod.py

```diff
@@ -1,26 +1,22 @@
 from __future__ import print_function, division
 
-from sympy.core.numbers import nan
+from sympy.core.numbers import nan, Integer
+from sympy.core.compatibility import integer_types
 from .function import Function
 
 
 class Mod(Function):
     """Represents a modulo operation on symbolic expressions.
-
     Receives two arguments, dividend p and divisor q.
-
     The convention used is the same as Python's: the remainder always has the
     same sign as the divisor.
-
     Examples
     ========
-
```

Comment:
Keep empty lines in the documentation unless you know for sure that they should be removed.

Reply:
Keep these lines. There should be empty lines around the headings. Also, in the above example, it really makes sense to separate the paragraphs with empty lines. So restore the file completely.